specification/keyvault/data-plane/Microsoft.KeyVault/preview/7.0/keyvault.json

```diff
@@ -0,0 +1,4793 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "title": "KeyVaultClient",
+    "description": "The key vault client performs cryptographic key operations and vault operations against the Key Vault service.",
+    "version": "2016-10-01"
```

Comment:
don't we need to change this? :)

Reply:
Good catch.  I had this change originally but I must have blown it away when I merged in updates from 2016-10-01